images/win/scripts/Installers/Install-DACFx.ps1

```diff
@@ -5,7 +5,6 @@
 
 Import-Module -Name ImageHelpers -Force
 
-$InstallerName = "DacFramework.msi"
-$InstallerUrl = "https://download.microsoft.com/download/f/1/9/f19eaee6-0728-4a0b-9755-9808acc8af0b/EN/x64/${InstallerName}"
+$InstallerUrl = "https://go.microsoft.com/fwlink/?linkid=2128142"
 
 Install-Binary -Url $InstallerUrl -Name $InstallerName
```

Comment:
You have removed `$InstallerName` but it is still used in 10 line

Reply:
thank you, I have added it back.